tests/fixtures/model_fixture.py

```diff
@@ -38,24 +38,22 @@ def upload_model_to_local_bucketfs(download_model) -> PurePosixPath:
 
 @pytest.fixture(scope="session")
 def upload_dummy_model_to_local_bucketfs() -> List[Tuple[str, PurePosixPath]] :
+    sub_dir = 'sub_dir'
     model_file_data_map = {
         "model_file1.txt": "Sample data in model_file1.txt",
         "model_file2.txt": "Sample data in model_file1.txt"}
 
-    models_metadata = []
     with tempfile.TemporaryDirectory() as tmpdir_name:
-        for sub_dir in ['sub_dir_1', 'sub_dir_2', 'sub_dir_3', 'sub_dir_4']:
-            model_path = PurePosixPath(
-                tmpdir_name,
-                bucketfs_operations.get_model_path(sub_dir, model_params.name))
-            bucketfs_location = LocalFSMockBucketFSLocation(model_path)
-
-            for file_name, content in model_file_data_map.items():
-                bucketfs_location.upload_string_to_bucketfs(
-                    str(PurePosixPath(tmpdir_name, file_name)), content)
-
-            models_metadata.append((sub_dir, str(model_path)))
-        yield models_metadata
+        model_path = PurePosixPath(
+            tmpdir_name,
+            bucketfs_operations.get_model_path(sub_dir, model_params.name))
+        bucketfs_location = LocalFSMockBucketFSLocation(model_path)
+
+        for file_name, content in model_file_data_map.items():
+            bucketfs_location.upload_string_to_bucketfs(
```

Comment:
In this test we mock the model load completely, so we don't need a localbucketfs mock. Not sure if we use this function somewhere else.

Reply:
Or, do I miss something?

Reply:
we need to provide a valid bucketfs connection for location in here:

https://github.com/exasol/transformers-extension/blob/24e653f2efd980d17cc66e4dfeb89a11576fb1c4/exasol_transformers_extension/udfs/sequence_classification_single_text_udf.py#L68

Reply:
Ah, you are right. Do we need files for that? Or, only a valid URI?

Reply:
valid URI, we can remove file uploading part